src/Xamarin.Android.Build.Tasks/Tasks/GetImportedLibraries.cs

```diff
@@ -37,7 +37,7 @@ public override bool Execute ()
 				return true;
 			}
 			// there could be ./AndroidManifest.xml and bin/AndroidManifest.xml, which will be the same. So, ignore "bin" ones.
-			ManifestDocuments = Directory.GetFiles (TargetDirectory, "AndroidManifest.xml", SearchOption.AllDirectories).Where (f => !Path.GetDirectoryName (f).EndsWith ("bin")).Select (p => new TaskItem (p)).ToArray ();
+			ManifestDocuments = Directory.GetFiles (TargetDirectory, "AndroidManifest.xml", SearchOption.AllDirectories).Where (f => !Path.GetDirectoryName (f).EndsWith ("bin") && !Path.GetDirectoryName (f).EndsWith ("manifest")).Select (p => new TaskItem (p)).ToArray ();
```

Comment:
Maybe this `Where` should be:
```csharp
.Where (f => {
    //This should evaluate to the directory name https://dotnetfiddle.net/EZLmBC
    var directory = Path.GetFileName (Path.GetDirectoryName (f));
    return directory != "bin" && directory != "manifest";
})
```

The way it is here, if a directory ended with `bin` or `manifest`, it would get skipped. Not sure if that is a real issue?

Reply:
I'll update the code. We need to skip both bin and manifest.

Reply:
It looks like the original would skip `foo_bin` and `foo_manifest`, too...